spacy/tests/lang/pt/test_noun_chunks.py

```diff
@@ -0,0 +1,221 @@
+from spacy.tokens import Doc
+import pytest
+
+
+# fmt: off
+@pytest.mark.parametrize(
+    "words,heads,deps,pos,chunk_offsets",
+    [
+        # determiner + noun
+        # um cachorro -> um cachorro
+        (
+            ["um", "cachorro"],
+            [1, 1],
+            ["det", "ROOT"],
+            ["DET", "NOUN"],
+            [(0, 2)],
+        ),
+        # two determiners + noun
+        # meu o pai -> meu o pai
+        (
+            ["meu", "o", "pai"],
+            [2, 2, 2],
+            ["det", "det", "ROOT"],
+            ["DET", "DET", "NOUN"],
+            [(0, 3)],
+        ),
+        # two determiners + noun
+        # todos essos caros -> todos essos caros
+        (
+            ["todos", "essos", "caros"],
+            [2, 2, 2],
+            ["det", "det", "ROOT"],
+            ["DET", "DET", "NOUN"],
+            [(0, 3)],
+        ),
+        # two determiners, one is after noun
+        # um irmão meu -> um irmão meu
+        (
+            ["um", "irmão", "meu"],
+            [1, 1, 1],
+            ["det", "ROOT", "det"],
+            ["DET", "NOUN", "DET"],
+            [(0, 3)],
+        ),
+        # two determiners + noun
+        # o meu pai -> o meu pai
+        (
+            ["o", "meu", "pai"],
+            [2, 2, 2],
+            ["det","det", "ROOT"],
+            ["DET", "DET", "NOUN"],
+            [(0, 3)],
+        ),
+        # relative pronoun
+        # A bicicleta essa está estragada -> A bicicleta
+        (
+            ['A', 'bicicleta', 'essa', 'está', 'estragada'],
+            [1, 4, 1, 4, 4],
+            ['det', 'nsubj', 'det', 'cop', 'ROOT'],
+            ['DET', 'NOUN', 'PRON', 'AUX', 'ADJ'],
+            [(0,2)]
+        ),
+        # relative subclause
+        #  o computador que comprou -> o computador
+        (
+            ['o', 'computador', 'que', 'comprou'],
+            [1, 1, 3, 1],
+            ['det', 'ROOT', 'nsubj', 'acl:relcl'],
+            ['DET', 'NOUN', 'PRON', 'VERB'],
+            [(0, 2), (2, 3)]
+        ),
+        # det + noun + adj
+        # O cachorro marrom  -> O cachorro marrom
+        (
+            ["O", "cachorro", "marrom"],
+            [1, 1, 1],
+            ["det", "ROOT", "amod"],
+            ["DET", "NOUN", "ADJ"],
+            [(0, 3)],
+        ),
+        # det + noun + adj plural
+        # As calças baratas  -> As calças baratas
+        (
+            ["As", "calças", "baratas"],
+            [1, 1, 1],
+            ["det", "ROOT", "amod"],
+            ["DET", "NOUN", "ADJ"],
+            [(0, 3)],
+        ),
+        # det + adj + noun
+        # Uma boa ideia -> Uma boa ideia
+        (
+            ['uma', 'boa', 'ideia'],
+            [2, 2, 2],
+            ["det", "amod", "ROOT"],
+            ["DET", "ADJ", "NOUN"],
+            [(0,3)]
+        ),
+        # multiple adjectives
```

Comment:
I also tested the `det + adj + noun + adj` case that didn't work so well in Spanish and it worked like a charm:

```
        # Uma boa gatinha parda -> Uma boa gatinha parda
        (
            ['uma', 'boa', 'gatinha', 'parda'],
            [2, 2, 2, 2],
            ["det", "amod", "ROOT", "amod"],
            ["DET", "ADJ", "NOUN", "ADJ"],
            [(0,4)]
        ),
```
No issues :)